R2API/ItemDrop/Catalog.cs

```diff
@@ -18,24 +19,13 @@ public static class Catalog {
                 if (!Loaded) {
                     foreach (var itemIndex in ItemCatalog.allItems) {
                         var itemDef = ItemCatalog.GetItemDef(itemIndex);
-                        foreach (var itemTag in itemDef.tags) {
-                            if (itemTag == ItemTag.Scrap) {
+                        if (itemDef.tier != ItemTier.NoTier && itemDef.pickupIconSprite != null && itemDef.pickupIconSprite.name != DropList.NullIconTextureName) {
```

Comment:
Why the check on Null Icons?

Reply:
There are a bunch of half finished / broken items in the game. Those items all have some combination of having their tier set as NoTier, they have no icon, or their icon is the TexNullIcon. Checking for these things was my way to get only the finished items for my lists. There used to be a lot more lists in this script by I've been slowly doing away with them, so this check used to play a much larger role.